### PR TITLE
PLANNER-560 - ignoring test until it gets fixed

### PR DESCRIPTION
**Pull request by sutaakar**
Minor improvements included.

@psiroky FYI

### CODE REVIEW

**Comment by ge0ffrey** [comment]
+1

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java [hunk 1]
```diff
@@ -315,7 +316,6 @@ public void testTerminateEarlyNotExistingSolver() throws Exception {
         SolverInstance instance = new SolverInstance();
         instance.setSolverConfigFile( SOLVER_1_CONFIG );
         instance.setStatus( SolverInstance.SolverStatus.NOT_SOLVING );
-        instance.setPlanningProblem( loadPlanningProblem( 5, 15 ) );
```
Comment (ge0ffrey) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java]: Why remove the plannign problem loading?

Comment (sutaakar) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java]: It isn't needed when terminating solver, so I removed it. For solver termination Kie server needs to know just solver status.

Comment (ge0ffrey) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java]: agreed

**Comment by psiroky** [comment]
Cherry-picked into 6.4.x:https://github.com/droolsjbpm/droolsjbpm-integration/commit/530b41f224798f98867e79eca52b429bcde5e1bf

**Pull request merged**